src/prover_calculi/superposition.ml

```diff
@@ -1614,6 +2711,45 @@ module Make(Env : Env.S) : S with module Env = Env = struct
   let condensation c =
     Util.with_prof prof_condensation condensation_rec c
 
+  let recognize_injectivity c =
```

Comment:
same as above wrt theory of injective funs

Reply:
So how should I rewrite this? Is it supported by already existing Zip code?